strapi/strapi
# feat(content-releases): Release Details page

### What does it do?

Add real content to the Release Details page, with the Release info and all the actions attached in a table 

### Why is it needed?

We need a way to edit the release (title, number of entries) and a table with all the actions 
The table for now should include the following columns:
 - Name (tbd: this will need to be the mainField or id)
 - ContentType
 - Locale

### How to test it?

You can go to the Releases page and create new Release
- inside a Collection in the CM select an entry and add it to the new Release
- then go to the Releases list page and click the release created, you can now see all the Release details and the actions
